### PR TITLE
Improve get_wd_search_results

### PR DESCRIPTION
**Pull request by Mystou**
Add an option to get_wd_search_results to return an dict of id and label.

### CODE REVIEW

File: wikidataintegrator/wdi_core.py [hunk 1]
```diff
@@ -368,8 +369,10 @@ def get_wd_search_results(search_string='', mediawiki_api_url=None,
                 raise WDSearchError('WD search failed')
             else:
                 for i in search_results['search']:
-                    id_list.append(i['id'])
-                    id_labels.append(i['label'])
+                    if dict_id_label:
+                        results.append({'id':i['id'],'label':i['label']})
+                    else:
+                        results.append(i['id'])
```
Comment (physikerwelt) [inline wikidataintegrator/wdi_core.py:375]: I guess you loose the label here.

Comment (Mystou) [reply wikidataintegrator/wdi_core.py:375]: No, id_labels was not used here. Only id_list was returned.

Comment (physikerwelt) [reply wikidataintegrator/wdi_core.py:375]: I see, `id_labels` was basically never read. Note that [pyCharm](https://www.jetbrains.com/help/resharper/UnusedVariable.html) discovers such issues automatically and suggests to remove code that has no effect.

**Comment by Mystou** [comment]
No news about this pull request?

**Pull request merged**